webui/react/src/shared/utils/data.test.ts

```diff
@@ -143,8 +143,8 @@ describe('Data Utilities', () => {
       { input: [ Symbol('bit'), 123 ], output: false },
       {
         input: [
-          new Set([ 'abc', 123, Symbol('xyz') ]),
-          new Set([ 'abc', 123, Symbol('xyz') ]),
```

Comment:
this tested ended up failing because `Symbol('xyz') !== Symbol('xyz')`. the whole point of Symbol is that separate instantiations are not considered equal, so it seems appropriate.

on a related note, it might make more sense for `isEqual(Symbol('xyz'), Symbol('xyz')) === false`